src/main.js

```diff
@@ -213,15 +213,24 @@ let load = Loader({
     }
   },
 
-  'expire-sentry': {
-    requires: ['cfg', 'sentryManager'],
-    setup: async ({cfg, sentryManager}) => {
+  'expire': {
```

Comment:
I may factor this a bit for tests, and yes, running in parallel makes sense.  But two Heroku dynos seems like overkill for two fairly short jobs that do similar things (expire stuff).  Is there an advantage?  I'm not sure what the Heroku costs are, either..